src/App.tsx

```diff
@@ -72,7 +72,7 @@ const MapContainer: React.FC<MapComponentProps> = ({cities}) => {
         latitude: 0,
         zoom: 1,
       }}
-      mapStyle="https://demotiles.maplibre.org/style.json"
+      mapStyle="/style.json"
```

Comment:
```suggestion
      mapStyle=process.env.PUBLIC_URL + "/style.json"
```
Please test if that works. It is better in case it is running from a subpath